README.md

```diff
@@ -53,6 +54,17 @@ npx simple-git-hooks
 
 ## `site` Setup
 
+### Local env vars
+
+Inside the `site` folder create a file called `.env.local` with the following content.
+
+```ini
+SENTRY_TOKEN=<sentry user auth token>
+SENTRY_UPLOAD=false
```

Comment:
```suggestion
SENTRY_TOKEN=<sentry user auth token>
SENTRY_UPLOAD=false # Enable/disable sending logs to Sentry
```

Reply:
this is just for source/sourcemaps uploading, capturing still work even with this `false`

Reply:
Ok, can we put this as the comment then - not obvious!